master/reference/node/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuring calico/node
+---
+
+The `calico/node` container is primarily configured through environment variables.
+
+
+## Environment Variables
+
+| Environment | Description                              | Schema | Default |
+| ------------- | -------- | ------ | ----- |
+| NODENAME | Node ID. | string | |
+| NO_DEFAULT_POOLS | Prevents calico from creating a default pool if one does not exist. | boolean | false |
+| HOSTNAME | Hostname. | string | |
+| IP | IP address which calico will bind to. | ip address | 10.0.0.1 |
+| IP6 | IPv6 address which calico will bind to. | ipv6 address | fed80::1 |
```

Comment:
If an IP address is specified here, the address is written in to the node resource configuration, and will override any value previously configured.

If no IP address is specified, and an address has not yet been configured in the node resource, we will not enable IPv6 routing.

If no IP address is specified and an IPv6 address has been previously configured in the node resource, we will use that address for IPv6 routing.